### PR TITLE
Implemented floordiv in sympy

### PR DESCRIPTION
**Pull request by arihantparsoya**
- This is fix for issue #11208 .
- Tests are added.
#### Output after this PR:

``` python
 >>> from sympy import *
>>> from sympy.abc import x,y
>>> x//y
floor(x/y)
>>> 3//2
1
```

### CODE REVIEW

**Comment by parsoyaarihant** [comment]
@asmeurer , can you review this?

File: sympy/core/expr.py [hunk 1]
```diff
@@ -308,6 +308,10 @@ def __lt__(self, other):
                 return sympify(dif.is_negative)
         return StrictLessThan(self, other, evaluate=False)
 
+    def __floordiv__(self,other):
```
Comment (leosartaj) [inline sympy/core/expr.py]: Leave space after ``,`` -> ``self, other``.

File: sympy/core/expr.py [hunk 2]
```diff
@@ -308,6 +308,10 @@ def __lt__(self, other):
...
+        from sympy.functions.elementary.integers import floor
+        return floor(self/other)
```
Comment (leosartaj) [inline sympy/core/expr.py]: ``floor(self / other)``

File: sympy/core/tests/test_expr.py [hunk 3]
```diff
@@ -1277,6 +1277,10 @@ def test_issue_5300():
     x = Symbol('x', commutative=False)
     assert x*sqrt(2)/sqrt(6) == x*sqrt(3)/3
 
+def test_floordiv():
+    from sympy.functions.elementary.integers import floor
+    assert x//y == floor(x/y)
```
Comment (leosartaj) [inline sympy/core/tests/test_expr.py]: ``x // y == floor(x / y)``

**Comment by leosartaj** [comment]
@parsoyaarihant Can you remove the unwanted commits?

File: sympy/core/tests/test_expr.py (hunk 3 above)
Comment (parsoyaarihant) [inline sympy/core/tests/test_expr.py]: @leosartaj thanks a lot.

**Comment by leosartaj** [comment]
Except from some formatting issues, this looks good. +1.

**Comment by asmeurer** [comment]
You should add `_sympifyit` and `call_highest_priority` decorators (see the other `__op__` methods on Expr), and define `__rfloordiv__`.

**Pull request closed without merging**